src/twilio/functions/sms/pause-resources.js

```diff
@@ -0,0 +1,23 @@
+exports.handler = function(context, event, callback) {
+  const twilioClient = context.getTwilioClient();
+  twilioClient.messages.create({
+      to: event.From,
+      from: event.To,
+      body:
+        "You have reached Crown Heights Mutual Aid. We are a network of neighbors helping neighbors. We are volunteers who give our time, pool our resources, and stand in solidarity with our neighbors in need.\n\nBetween June 21st and July 19th 2020 we will not be able to take new calls. We are using this time to call back the thousands of neighbors who called us before June 21st.\n\nIf you called us before June 21st, your call is recorded. We are prioritizing our Crown Heights neighbors at this time, but will do our best to call everyone back. We are planning to re-open our phone line on July 19th 2020."
+    })
+    .then(function() {
+      twilioClient.messages.create({
+          to: event.From,
+          from: event.To,
+          body:
+            "In the meantime we can provide other resources in Brooklyn. We will have a list on our website at crownheightsmutualaid.com. We will post updates on facebook https://www.facebook.com/groups/496603171016990 and instagram, @crownheightsaid.\n\nYou can visit https://mutualaid.nyc/resources-groups to find a Mutual Aid group in your neighborhood. You can also text FOOD to 726879 to make a reservation to pick up food at a pantry within walking distance of your home, in any borough.\n\nStay safe!"
```

Comment:
since both these messages might be used for #96, i'd love to find a way to put these in the /lib/strings directory without requiring all the dependencies. 

what if you created a `twilio.json` file in `src/lib/strings/locales/en` that is the same format as the other ones? You can import json as an object in node, so it should be pretty nice except the ugly import which will look something like:
`const twilioStrings = require(../../../lib/strings/locales/en/twilio.json`

Reply:
Added this in #96 !
https://github.com/crownheightsaid/mutual-aid-app/pull/96/files#diff-ff63639130e2c980af1103809f45ce69R14

Reply:
cool thnx joe!